.env.development

```diff
@@ -1,3 +1,5 @@
-REACT_APP_OIDC_AUTHORITY="https://api.hel.fi/sso-test/"
-REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile-ui"
-REACT_APP_PROFILE_GRAPHQL="https://profiili-api.test.kuva.hel.ninja/graphql/"
+REACT_APP_OIDC_AUTHORITY="https://tunnistamo.dev.hel.ninja/"
+REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile-ui-dev"
+REACT_APP_PROFILE_AUDIENCE="https://api.hel.fi/auth/helsinkiprofiledev"
+REACT_APP_PROFILE_GRAPHQL="https://helsinki-profile-api-dev.agw.arodevtest.hel.fi/graphql/"
+REACT_APP_OIDC_SCOPE="openid profile https://api.hel.fi/auth/helsinkiprofiledev"
```

Comment:
Is this (and the corresponding `public/env-config.js` change) somehow related to the apollo client update, as these changes are now in the same commit? I don't understand the relevance.

Reply:
Previous, very old, configs did not provide a login flow that results in getting Verified Personal Information. Time and need to update them to use OpenShift.

Reply:
But could the config be changed in its own commit, if it's not tightly related to the apollo client update?

Reply:
Sliced and diced the file (+ auto-generated env-config.js) to another commit.